spec/03-plugins/03-http-log/01-log_spec.lua

```diff
@@ -293,6 +293,7 @@ for _, strategy in helpers.each_strategy() do
 
         if #body.entries == 1 then
           assert.same("127.0.0.1", body.entries[1].client_ip)
+          assert.not_nil(body.entries[1].workspace_name)
```

Comment:
Assert the workspace name in one of the test and please make sure to test for default and non-default workspaces.

Reply:
I see. But the Kong OSS only supports the default workspace. So do we need to test the non-default workspace?

Reply:
Ah. Can you please add a test for this in the other repo (after cherry-picking it)?

Reply:
Sure. I will add a test case in the ee.